server/dummyData/dummySurveys.js

```diff
@@ -249,38 +249,10 @@ module.exports = [
     dateClosed: 1576851113,
     anonymous: true,
     recipients: [
```

Comment:
why was this needed?

Reply:
We only had one survey that didn't invite everyone which was why I didn't see the bug until today, and the title of the survey was "people team engagement survey" so seemed like it would be more sensible to just invite the people team.

Reply:
thats great, thank you!